egs/librispeech/WSASR/README.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+This is a weakly supervised ASR recipe for the LibriSpeech (clean 100 hours) dataset. We training a
+conformer model using BTC/OTC with transcripts with synthetic errors. In this README, we will describe
```

Comment:
Could you also note down what is BTC and OTC?

Reply:
Updated.